lms/static/sass/shared/_header.scss

```diff
@@ -357,7 +357,7 @@ header.global {
 // ====================
 
 // CASE: marketing/course discovery
-header.global-new {
+.header-global {
```

Comment:
Thanks for diagnosing this, @AlasdairSwan. I've attempted to revert this back, so let me know if this is what you had in mind. It is a pity to have ``header-global-new`` now that we also have a new pattern library header, but at this point it seems cleaner to leave it as is. Hopefully we can rationalize these header classes in the future. Let me know what you think.